app/views/placements/schools/placements/_details.html.erb

```diff
@@ -24,6 +24,42 @@
       <% end %>
     <% end %>
   <% end %>
+  <% summary_list.with_row do |row| %>
+    <% row.with_key(text: t(".attributes.placements.academic_year")) %>
+    <% row.with_value do %>
+        <% placement.academic_year.name %>
+    <% end %>
+  <% end %>
+    <% summary_list.with_row do |row| %>
+      <% row.with_key(text: t(".attributes.placements.terms")) %>
+      <% row.with_value do %>
+        <%= placement.term_names %>
+      <% end %>
+      <% row.with_action(
+      text: t(".change"),
+      href: edit_attribute_path(:terms),
+      visually_hidden_text: t(".attributes.placements.terms"),
+    ) %>
+  <% end %>
+  <% summary_list.with_row do |row| %>
+    <% row.with_key(text: t(".attributes.placements.academic_year")) %>
+    <% row.with_value do %>
+        <% placement.academic_year.name %>
+    <% end %>
+  <% end %>
+  <% if placement.terms.any? %>
+      <% summary_list.with_row do |row| %>
+        <% row.with_key(text: t(".attributes.placements.terms")) %>
+        <% row.with_value do %>
+          <%= placement.term_names %>
+        <% end %>
+        <% row.with_action(
+        text: t(".change"),
+        href: edit_attribute_path(:terms),
+        visually_hidden_text: t(".attributes.placements.terms"),
+      ) %>
+      <% end %>
+    <% end %>
```

Comment:
Something has gone wrong with your rebase. You'll need to remove this duplicated code.